dcos-metrics-collector.go

```diff
@@ -39,7 +44,7 @@ type AgentConfig struct {
 // Config ...
 type Config struct {
 	HTTPProfiler  bool   `yaml:"http_profiler"`
-	KafkaProducer bool   `yaml:"kafka_producer"`
+	KafkaProducer bool   `yaml:"kafka_producer_enabled"`
```

Comment:
I like that idea, though we should probably do this in a later PR so we don't over reach what we refactor here.